build/upgrade_highlightjs.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+# Run with specific branch/tag (e.g. ./upgrade_highlightjs.sh 10.6.0) or with no arguments to use master
+
+# TODO: This list of languages should be made more dynamic based on the file extensions Preview supports
+HIGHLIGHTJS_LANGUAGES="actionscript dos c cpp cmake csharp css diff erb groovy haml java javascript json less makefile ocaml php perl properties python ruby scss scala sml sql bash vim yaml"
+TEXT_BASE_PATH="src/third-party/text"
+TEXT_STATIC_ASSETS_BRANCH=${1:-master}
+TEXT_STATIC_ASSETS_VERSION=$(./build/current_version.sh)
+TEXT_STATIC_ASSETS_PATH="${TEXT_BASE_PATH}/${TEXT_STATIC_ASSETS_VERSION}"
+
+build_custom_highlightjs() {
+    echo "-----------------------------------------------------------------------------------"
+    echo "Cloning higlightjs repo at branch: $TEXT_STATIC_ASSETS_BRANCH..."
+    echo "-----------------------------------------------------------------------------------"
+    rm -rf ./highlight.js
+    git clone git@github.com:highlightjs/highlight.js.git --depth 1 --single-branch --branch ${TEXT_STATIC_ASSETS_BRANCH} || return 1
+
+    echo "-----------------------------------------------------------------------------------"
+    echo "Building higlightjs for browser for languages ${HIGHLIGHTJS_LANGUAGES}..."
+    echo "-----------------------------------------------------------------------------------"
+    cd ./highlight.js
+    yarn install || return 1
```

Comment:
Should we use `--frozen-lockfile` here to avoid any potential issues with dependency drift?